Sources/BloomFilterWrapper/BloomFilterWrapper.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  BloomFilterWrapper.swift
+//
+//  Copyright © 2023 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+@_implementationOnly import BloomFilterObjC
```

Comment:
Make sure that `BloomFilterObjC` is not exposed externally.

The underscore in `@_implementationOnly` means it's subject to future changes, so we may need to revisit this if the Swift specification changes, but it's really convenient to ensure we're not exposing the underlying statically linked library..

Reply:
I think this is fine - `@_implementationOnly` is known to be used in some fairly major production SDKs, so I have a hard time seeing anything happen to it without a suitable replacement.